docs/_source/guides/programmatic_labeling_with_rules.ipynb

```diff
@@ -49,7 +51,7 @@
     "### Weak labeling using the UI\n",
     "\n",
     "Since version 0.8.0 you can find and define rules directly in the UI. \n",
-    "The [Define rules mode](../../reference/webapp/pages.html#modes) is found in the right side bar of the [Dataset page](../../reference/webapp/pages.html#dataset).\n",
+    "The [Define rules mode](../reference/webapp/pages.html#modes) is found in the right side bar of the [Dataset page](../reference/webapp/pages.html#dataset).\n",
```

Comment:
This mode is actually called `Weak labeling`, so we should probably correct this. I see that in the page the link points to 'labeling' is written with double l, would that be corrected with your other PR?
```suggestion
    "The [weak labeling mode](../reference/webapp/pages.html#modes) to define rules is found in the right side bar of the [Dataset page](../reference/webapp/pages.html#dataset).\n",
```

Reply:
I didn't update filenames that used double L in the other PR, as then links from outside of Argilla to those pages will break.